extractphpfiles.py

```diff
@@ -2,10 +2,13 @@
 
 folder = "wordpress/"
 ext = "*.php"
-for filename in [y for x in os.walk(folder) for y in glob.glob(os.path.join(x[0], ext))]:
-    with open(filename, "r") as f:
-        content = f.read()
-    if content.count("<?php") > 1:
-        continue
-    with open("phpfiles/{}".format(os.path.basename(filename)), "w") as f:
-        f.write(content.replace("<?php", ""))
+for d in os.walk(folder):
+    for f in d[2]:
+        if os.path.splitext(f)[1] == ".php":
+            filename = os.path.join(d[0], f)
+            with open(filename, "r") as f:
+                content = f.read()
+            if content.count("<?php") > 1:
+                continue
+            with open("phpfiles/{}".format(os.path.basename(filename)), "w") as f:
+                f.write(content.replace("<?php", ""))
```

Comment:
What about the trailing `>`?

Reply:
For some reason all PHP files in wordpress omit the closing `>?`, so I didn't bother adding this. Though if we ever use another base for extract php files from, we would have to remove this as well.

Reply:
Weird, but OK!